src/components/MySpace/MySpace.module.scss

```diff
@@ -0,0 +1,18 @@
+@import 'styles/uswdsDependencies';
+@import 'styles/mvp/mixins';
+@import 'styles/mvp/settings';
+
+.mySpace {
+  :global {
+    h2 {
```

Comment:
since `h2` is an html element selector (and not a CSS class) you don't need to wrap it in the `:global` scope